pkg/collector/corechecks/containers/docker.go

```diff
@@ -124,7 +125,7 @@ func (d *DockerCheck) countAndWeightImages(sender aggregator.Sender) error {
 
 // Run executes the check
 func (d *DockerCheck) Run() error {
-	sender, err := aggregator.GetSender(d.ID())
+	sender, err := d.updateCappedSender()
```

Comment:
Does that mean we always use it? If so we always use the custom `Rate` method. Could we only use the CappedSender when `capped_metrics` is set instead?